tests/typ/math/math-module.typ

```diff
@@ -0,0 +1,45 @@
+// Tests the "math" module has the elems
```

Comment:
This test is unnecessary imo, since it exactly the mirror the list in mod.rs. It doesn't add any new information.